lib/log.go

```diff
@@ -17,22 +16,26 @@ import (
 // LogMethods extends a lib.Instance with business logic for working with lists
 // of dataset versions. think "git log".
 type LogMethods struct {
-	inst *Instance
+	d dispatcher
 }
 
-// CoreRequestsName implements the Requets interface
-func (m LogMethods) CoreRequestsName() string { return "log" }
+// Name returns the name of this method group
+func (m LogMethods) Name() string {
+	return "log"
+}
 
-// NewLogMethods creates a LogMethods pointer from either a repo
-// or an rpc.Client
-func NewLogMethods(inst *Instance) *LogMethods {
-	return &LogMethods{
-		inst: inst,
+// Attributes defines attributes for each method
+func (m LogMethods) Attributes() map[string]AttributeSet {
+	return map[string]AttributeSet{
+		"history":        {AEHistory, "POST"},
+		"entries":        {AEEntries, "POST"},
+		"rawlogbook":     {denyRPC, ""},
+		"logbooksummary": {denyRPC, ""},
```

Comment:
read the main pr comment for why I think these are good method names:
https://github.com/qri-io/qri/pull/1730#issue-604927842